packages/website/src/layout/ElementWrapper.tsx

```diff
@@ -1,21 +1,32 @@
-import * as React from 'react';
-import _merge from 'lodash/merge';
-import _omit from 'lodash/omit';
-import queryString from 'query-string';
-
-import { Provider, ToastManager } from 'bumbag';
-import defaultTheme from '../theme';
-import { themeMap } from '../utils/theme';
+import React, { useState, useEffect } from 'react';
+import _isEmpty from 'lodash/isEmpty';
+import { getTheme, APPS } from '@localz/localz-bumbag-theme';
+import { Provider, ToastManager, Spinner, Container } from 'bumbag';
 
 export default function ElementWrapper(props: { element: React.ReactNode }) {
+  const [theme, setTheme] = useState({});
   const { element } = props;
-  const themeName = (queryString.parse(typeof window !== 'undefined' ? window.location.search : '') || {}).theme;
-  const targetTheme = themeMap[themeName] || {};
-  const theme = _merge(defaultTheme, _omit(targetTheme, 'SideNav', 'PageWithSidebar'));
 
+  useEffect(() => {
+    const getLocalzTheme = async () => {
+      try {
+        const localzTheme = await getTheme({ projectId: 'projectId', isDev: true, app: APPS.dashboard });
```

Comment:
Will `isDev` be sufficient here? Assuming the site deployment will pretend to reach out to dev config api to load in a nonexistent project's dashboard theme resulting in just the defaults being returned?

Reply:
good question 🤔  it's probably better to point to prod config api if env is prod